src/types.d.ts

```diff
@@ -1,8 +1,31 @@
-export interface Rule {
+export type Rule = {
+	engine: string;
+	sourcepackage: string;
 	name: string;
 	description: string;
-	sourcepackage: string;
 	categories: string[];
 	rulesets: string[];
 	languages: string[];
 }
+
+export type NamedPaths = {
```

Comment:
Class names that are pluralized bother me. Could we rename this something like `NamedPathsDescriptor` so it's clear that it's a single object that describes multiple named paths?

Reply:
Took a different approach: renamed NamedPaths to PathGroup.  Seems appropriate, since this is really a grouping of paths, and groups have names.